app/page.tsx

```diff
@@ -26,6 +27,13 @@ export default async function Home() {
   const discussions = await fetchGithubDiscussion(5);
   const startDate = parseISO(env.NEXT_PUBLIC_ORG_START_DATE);
 
+  const topContributors = [
```

Comment:
use proper variable names. is this still topContributors? or highlight contributors?